pkg/library/workspace/helper.go

```diff
@@ -0,0 +1,43 @@
+//
+// Copyright (c) 2019-2022 Red Hat, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package workspace
```

Comment:
I'm not really sure about this package name or file name...
We already have `/pkg/controllers/workspace` and there's also another file called `helper.go` for the flatten package.

Reply:
Why not name the package e.g. defaults or workspacedefaults

Reply:
I went with defaults as it seems more generic/could potentially be reused more easily in the future. However, I'm not yet sure about the filename `helper.go`, maybe it should just be called `workspace.go`?